css/order online.css

```diff
@@ -0,0 +1,407 @@
+
+:root {
+    --heading-font: 'Questrial', sans-serif;
+    --button-font: 'Raleway', sans-serif;
+    --body-font:'Roboto', sans-serif;
+    --main-color: #fff;
+    --secondary-color: #000;
+    --store-page-box-color: rgb(139, 139, 139);
+    --background-color-1: #f5e8ff;
+    --background-color-2 :#E6FAC0;
+    --footer-color: #304B2A;
+    --button-hover: #5E7D1F;
+}
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    list-style-type: none;
+    text-decoration: none;
+} 
+button {
+    font-family: var(--button-font);
+    background-color: transparent;
+}
+a {
+    color:var(--secondary-color);
+}
+img {
+    width: 100%;
+    object-fit: cover;
+}
+h1, h2 {
+    font-family: var(--heading-font);
+    font-weight: 100;
+    line-height: 2.3rem;
+    font-size: 2rem;
+}
+body {
+    font-family: var(--body-font);
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    background-color: var(--background-color-2);
+}
+header {
+    grid-column: span 12;
+    background-color: var(--main-color);
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+}
+.header-user-icon {
+    grid-column: 6/8;
+    width: auto;
+    height: 100%; 
+    display: inline;
+}
+.header-logo {
+    grid-column: span 6;
+    font-family: var(--heading-font);
+    text-align: left;
+    font-weight: 400;
+    font-size: 1.25rem;
+    padding-left: 0.5rem;
+}
+.header-hamburger-icon {
+    grid-column: 11/ 12;
+    width: auto;
+    height: 100%; 
+} 
+main {
+    grid-column: span 12;
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    grid-template-rows: repeat(6, auto);
+}
```

Comment:
I like what you've done with this page 🌟 
I think in this case it might be easier to split the main area into 3 columns of unequal size, and only have one row. And then inside each column, you might not need to use CSS Grid. For example, the central section can probably be achieved quite easily with flexbox: wrap each product/item block into a div, and align the image + text + price inside that div into a row using flexbox and padding. Probably easier than dealing with lots of columns.